src/ACTSMergeRelationCollections.cxx

```diff
@@ -0,0 +1,39 @@
+#include "ACTSMergeRelationCollections.hxx"
+
+// edm4hep
+#include <edm4hep/MCRecoTrackerHitPlaneAssociation.h>
+
+
+DECLARE_COMPONENT(ACTSMergeRelationCollections)
+
+
+ACTSMergeRelationCollections::ACTSMergeRelationCollections(const std::string& name, ISvcLocator* svcLoc) : MultiTransformer(name, svcLoc, {
+		KeyValue("InputCollection1", "Collection1"),
+		KeyValue("InputCollection2", "Collection2"),
+		KeyValue("InputCollection3", "Collection3"),
+		KeyValue("InputCollection4", "Collection4"),
+		KeyValue("InputCollection5", "Collection5"),
+		KeyValue("InputCollection6", "Collection6") },
+	      { KeyValue("OutputCollection", "MergedCollection") }) {}
+
+std::tuple<edm4hep::MCRecoTrackerHitPlaneAssociationCollection> ACTSMergeRelationCollections::operator()(
+		const DataWrapper<edm4hep::MCRecoTrackerHitPlaneAssociationCollection>& col1,
+		const DataWrapper<edm4hep::MCRecoTrackerHitPlaneAssociationCollection>& col2,
+		const DataWrapper<edm4hep::MCRecoTrackerHitPlaneAssociationCollection>& col3,
+		const DataWrapper<edm4hep::MCRecoTrackerHitPlaneAssociationCollection>& col4,
+		const DataWrapper<edm4hep::MCRecoTrackerHitPlaneAssociationCollection>& col5,
+		const DataWrapper<edm4hep::MCRecoTrackerHitPlaneAssociationCollection>& col6) const{
+	// Initialize new collection
+	edm4hep::MCRecoTrackerHitPlaneAssociationCollection mergedCollection;
```

Comment:
```suggestion
	edm4hep::TrackerHitSimTrackerHitLinkCollection mergedCollection;
```
`edm4hep::MCRecoTrackerHitPlaneAsssociation` has been removed in https://github.com/key4hep/EDM4hep/pull/331 and `Associations` have undergone some renaming in https://github.com/key4hep/EDM4hep/pull/341

For this part the same applies as for the tracker hit merging; I would make the returned collection a subset collection.

Reply:
These changes are present in `v00-99` (and `v00-99-01`) of EDM4hep. Maybe we can try to put together a new image with these versions. I am not entirely sure how well a "mixing and matching" of different EDM4hep versions in the same stack will work at runtime. It might work, but it could also end in complete chaos.

This is not the most important change though. Effectively it will be an exercise of search and replace once the version has been switched, resp. when going to `v00-99` (or `v00-99-01`) everything will still work, you will just get a few deprectation warnings.